src/test/FuzzerImpl.cpp

```diff
@@ -768,6 +786,19 @@ TransactionFuzzer::initialize()
                     PublicKey issuer;
                     FuzzUtils::setShortKey(issuer, j);
 
+                    // Allow trust for any accounts whose parameters require it.
```

Comment:
This change doesn't make a lot of sense to me for a variety of reasons:
1. If `issuerAccount.flags == 0` then `trustLine.flags != 0` when the trust line is created (check how `ChangeTrust` works), so perhaps you actually want to set them to 0.
2. This is implicitly assuming that all the trust lines for a single asset should have the same flags.

Reply:
> This is implicitly assuming that all the trust lines for a single asset should have the same flags.

Same question as below -- I'd been planning to take that further step toward increased variety in a follow-up PR as I thought it would be clearer, but I might be misjudging that.

Reply:
> If issuerAccount.flags == 0 then trustLine.flags != 0 when the trust line is created (check how ChangeTrust works), so perhaps you actually want to set them to 0.

Aha -- I think I'm gather that `AUTHORIZED_FLAG` might already be set (if the issuer doesn't have `AUTH_REQUIRED` set) and `TRUSTLINE_CLAWBACK_ENABLED_FLAG` might already be set (if the issuer has `AUTH_CLAWBACK_ENABLED` set).

One option would be to make `mAllowTrustFlags` `optional`, being set only if we want the trust line to have flags other than the ones with which it was created as a consequence of the issuer's account flags.  That'd make the initialization of `accountParameters` more verbose, but it would also make it easy to eyeball exactly which trustline flags _weren't_ going to be in the same state after setup that they were in immediately after their creations.  I think that leads me to favor doing that, but am open to other preferences.